windows/devices/uwb/CMakeLists.txt

```
@@ -1,3 +1,46 @@
+add_library(libuwbcxadapter STATIC "")
+
+set(LIBUWBCXADAPTER_DIR_PUBLIC_INCLUDE ${CMAKE_CURRENT_LIST_DIR}/include)
+set(LIBUWBCXADAPTER_DIR_PUBLIC_INCLUDE_PREFIX ${LIBUWBCXADAPTER_DIR_PUBLIC_INCLUDE}/windows/devices/uwb)
+
+target_sources(libuwbcxadapter
+    PRIVATE
+        ${CMAKE_CURRENT_LIST_DIR}/UwbCxAdapter.cxx
+        ${CMAKE_CURRENT_LIST_DIR}/UwbAppConfiguration.cxx
+    PUBLIC
+        ${LIBUWBCXADAPTER_DIR_PUBLIC_INCLUDE_PREFIX}/UwbCxAdapter.hxx
+        ${LIBUWBCXADAPTER_DIR_PUBLIC_INCLUDE_PREFIX}/UwbAppConfiguration.hxx
```

Please make sure these are sorted in ascending order for consistency. Same goes for any other CMakeLists.txt files that have added files.

```suggestion
        ${CMAKE_CURRENT_LIST_DIR}/UwbAppConfiguration.cxx
        ${CMAKE_CURRENT_LIST_DIR}/UwbCxAdapter.cxx
    PUBLIC
        ${LIBUWBCXADAPTER_DIR_PUBLIC_INCLUDE_PREFIX}/UwbAppConfiguration.hxx
        ${LIBUWBCXADAPTER_DIR_PUBLIC_INCLUDE_PREFIX}/UwbCxAdapter.hxx
```